File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_4/exercice03.c
#include <stdio.h>
int N_maj,N_let;
char T[10];
int main(){
    for(int i=0;i<10;i++){
        printf("saisir le caractere %d : ",i+1);
        scanf(" %c",&T[i]);
        if(T[i]<='Z' && T[i]>='A')
            N_maj++;
        else if(T[i]<='z' && T[i]>='a')
            N_let++;
    }
    printf("le nombre de majuscules est : %d\n",N_maj);
    printf("le nombre de lettres est : %d\n",N_let);
    return 0;
}

File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_4/exercice03.py
T=[None for i in range(10)]
N_maj=0
N_let=0
for i in range(0,10):
    T[i]=str(input(f"saisir le caractere {i+1}: "))[0]
    if(T[i]>='A' and T[i]<='Z'):
        N_maj+=1
    elif(T[i]>='a' and T[i]<='z'):
        N_let+=1
print("le nombre de majuscules est : ",N_maj)
print("le nombre de lettres est : ",N_let)